.bacon.yml

```diff
@@ -0,0 +1,50 @@
+test_suites:
+    - name: lint
+      script_path: /root/okta/okta-signin-widget/scripts
+      sort_order: '1'
+      timeout: '60'
+      script_name: lint
+      criteria: MERGE
+      queue_name: small
+    - name: unit-karma
+      script_path: /root/okta/okta-signin-widget/scripts
+      sort_order: '2'
+      timeout: '10'
+      script_name: unit
+      criteria: MERGE
+      queue_name: small
+    - name: unit-jest
+      script_path: /root/okta/okta-signin-widget/scripts
+      sort_order: '3'
+      timeout: '10'
+      script_name: jest
+      criteria: MERGE
+      queue_name: small
+    - name: testcafe
+      script_path: /root/okta/okta-signin-widget/scripts
+      sort_order: '4'
+      timeout: '10'
```

Comment:
We'll want to bump these up a bit, as these tests take longer than the timeout specified here.